### DISCUSSION TITLE
Error when running PercentageFeatureSet with features set as a list

### DISCUSSION THREAD

**Discussion by NicoHadas** [satijalab/seurat#8521] [Q&A] [answered]
Hello,

Whenever I run PercentageFeatureSet with a list of ribosomal genes I get the following error:

`S70_1[["percent.ribo"]] <- PercentageFeatureSet(S70_1, features = ribo_list)`
Error in h(simpleError(msg, call)) : 
  error in evaluating the argument 'x' in selecting a method for function 'colSums': subscript out of bounds

Here is an example of my list

ribo_list
 [1] "Fau"        "Mrpl13"     "Rpl10"      "Rpl10a"     "Rpl10l"     "Rpl11"      "Rpl12"      "Rpl13"      "Rpl13a"     "Rpl14"      "Rpl15"      "Rpl17"      "Rpl18"      "Rpl18a"     "Rpl19"      "Rpl21"

Any advice would be appreciated.

**Comment by samuel-marsh** [comment, marked as answer]
Have you verified that all of those features are present in object?

`all(ribo_list %in% Features(S70_1))`
